services/net/transcription/TranscriptionManager.cs

```diff
@@ -228,14 +228,21 @@ private async Task UpdateTranscriptionAsync(ContentModel content)
         // TODO: Handle different storage locations.
         // Remote storage locations may not be easily accessible by this service.
         var path = content.FileReferences.FirstOrDefault()?.Path;
-        var safePath = Path.Join(_options.VolumePath, path.MakeRelativePath());
+        var safePath = Path.Join(_options.FilePath, path.MakeRelativePath());
```

Comment:
This should be `_options.VolumePath`.  The configuration is to map the volume.  The path of the file is everything after the volume path.

Reply:
There should be no `_options.FilePath` in the source code.